descriptors/ModuleDescriptor-template.json

```diff
@@ -107,7 +123,11 @@
           "methods": [
             "POST"
           ],
-          "modulePermissions": [],
+          "modulePermissions": [
```

Comment:
For Post tenant API I don't think that we need this permissions

Reply:
updated permission by adding user's module permissions.